### PR TITLE
network separation and integrated

### PR DESCRIPTION
**Pull request by zkn111**
Change-Id: I464c8384147b69403bb06cb0897610215f14eb9c

network separation and integrated
@PengJiyuan

### CODE REVIEW

**Comment by zkn111** [comment]
dashboard:
Delete ‘create network’： the network type

File: modules/network/model.jsx [hunk 1]
```diff
@@ -384,7 +390,7 @@ class Model extends React.Component {
         router: element.router ?
           <span>
             <i className="glyphicon icon-router"/>
-            <a data-type="router" href={'/dashboard/router/' + element.router.id}>{element.router.name || '(' + element.router.id.substr(0, 8) + ')'}</a>
+            <a data-type="router" href={'/admin/router/' + element.router.id}>{element.router.name || '(' + element.router.id.substr(0, 8) + ')'}</a>
```
Comment (PengJiyuan) [inline modules/network/model.jsx]: 从dashboard直接跳到admin？万一没有admin权限怎么办？

File: modules/subnet/pop/create_subnet/index.js [hunk 2]
```diff
@@ -111,23 +94,18 @@ function pop(obj, parent, callback) {
...
-              refs.select_network.setState({
-                data: [res],
-                value: res.id,
-                clicked: false
-              });
-              refs.btn.setState({
-                disabled: false
-              });
+        case 'input_network':
+          if(testAddr.test(netState.value) && networkState !== '') {
```
Comment (PengJiyuan) [inline modules/subnet/pop/create_subnet/index.js:98]: refs.btn.setState({
  disabled: !(testAddr.test(netState.value) && networkState !== '')
});

File: modules/network/model.jsx [hunk 3]
```diff
@@ -370,7 +370,13 @@ class Model extends React.Component {
         title: __.physical_network,
         content: networkType === 'flat' ? item['provider:physical_network'] : '-'
       });
+    } else if (networkType === 'vxlan') {
```
Comment (PengJiyuan) [inline modules/network/model.jsx:373]: content: networkType === 'vlan' ? (item['provider:segmentation_id'] ? item['provider:segmentation_id'] : '-') : (item['provider:physical_network'] ? item['provider:physical_network'] : '-')

**Pull request closed without merging**